sys/suit/handlers_command_seq.c

```diff
@@ -316,13 +322,16 @@ static int _dtv_fetch(suit_manifest_t *manifest, int key,
         return res;
     }
 
+    LOG_INFO("Updating flag\n");
```

Comment:
```suggestion
    LOG_DEBUG("Updating flag\n");
```

Reply:
Leftover debug logging :scream: I'm going to remove it, not enough content in this.